### ISSUE TITLE
Chutzpah configuration with V S TFS 2015

### ISSUE THREAD

**Issue by em-rkumar** [mmanela/chutzpah#458]
Hi,

I tried to configure , but all the time getting failed with an error , No Test Found.

**Comment by mmanela** [comment]
Please provide more information so I may be able to help you. Give details on what you tried and the steps you took

**Comment by csnate** [comment]
I may be able to provide some more details here as I believe I'm experiencing the same issue.

We recently upgraded our TFS instance to TFS 2015 (udpate 2).  Our existing XAML build was completing successfully, except in the output logs it would have the following:

Test Run Completed. 0 tests executed.
No test found. Make sure that installed test discoverers & executors, platform & framework version settings are appropriate and try again.

After wasting a couple of hours thinking this was our C# unit tests (doh!) I realized that this error was coming from our Chutzpah JS unit tests (QUnit). I followed the instructions outlined [here](https://blogs.msdn.microsoft.com/visualstudioalm/2012/07/09/javascript-unit-tests-on-team-foundation-service-with-chutzpah/) and setup the Chutzpah assemblies again, thinking there must have been something wrong in the upgrade.  However, I'm still getting this error in the build.

Not sure if TFS 2105 requires something new?

**Comment by mmanela** [comment]
Did you try to take the Chutzpah nuget package and include *all* the files ?

**Comment by csnate** [comment]
Initially, I (thought I) included all of the files except the following:

- [Content_Types].xml
- Chutzpah.vs2012.pkgdef
- EULA.rt
- extension.vsixmanifest

Then, when I went to add those files I realized that none of the assemblies were included in my initial commit since TFS won't include assemblies/executables unless intentionally added.  I've included everything now and it's working correctly again.  From my perspective, I think this can be closed.
